library/src/include/kernel_launch.h

```diff
@@ -19,8 +19,8 @@
 struct DeviceCallIn
 {
     TreeNode *node;
-    void *bufIn[2];
-    void *bufOut[2];
+    void *bufIn[2];//TODO, bufIn[1] is never used
+    void *bufOut[2];//TODO, bufOut[1] is never used
```

Comment:
actually when we support planar formats, the [1] indices would be used, just not now